code/modules/projectiles/projectile/bullets/shotgun.dm

```diff
@@ -43,6 +43,8 @@
 	light_color = LIGHT_COLOR_FIRE
 	damage_type = BURN
 	damage = 12 //slight damage on impact
+	wound_bonus = 60//Most wounds come from fire stacks, but this is insult to injury. :)
```

Comment:
```suggestion
	wound_bonus = 48//Most wounds come from fire stacks, but this is insult to injury. :)
```
60 wound bonus would make this deal stuff the likes of crit pierces through PA. Not good for an easily printable ammotype

Reply:
Considering how rare the flamethrower is, as it's exclusive to NCR more-or-less, I'm going to argue otherwise.